GPL/gpl.gradle

```diff
@@ -45,6 +46,11 @@ ext.getCurrentPlatformName = {
 			return 'osx64'
 		}
 	}
+	else if (osName.startsWith("Mac OS X")) {
+		if (isAARCH_64) {
```

Comment:
Shouldn't this be an "else if (isAARCH_64) within the previous "else if (osName.startsWith("Mac OS X"))"?